app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if file_to_move != "".join(destination):
+        if not os.path.exists(file_to_move):
+            raise FileNotFoundError
+        path = ""
+        path = os.path.join(path, *destination[:-1])
+        if path:
+            os.makedirs(path, exist_ok=True)
+        path = os.path.join(path, destination[-1])
```

Comment:
You don't need to initialize `path` with an empty string on line 11. You can directly assign the result of `os.path.join` to it on line 12. (Code Efficiency issue)